components/09-nautobot/base/nautobot/templates/configmap.yaml

```diff
@@ -581,7 +581,7 @@ data:
     SOCIAL_AUTH_OIDC_OIDC_ENDPOINT = "http://dexidp.local"
     SOCIAL_AUTH_OIDC_USERNAME_KEY = 'name'
     SOCIAL_AUTH_OIDC_KEY = 'nautobot'
-    SOCIAL_AUTH_OIDC_SECRET = 'verysecret'
+    SOCIAL_AUTH_OIDC_SECRET = open("/opt/nautobot/dex_client_secret").read()
```

Comment:
Technically this leaks a file descriptor

Reply:
cpython would normally close it during garbage collection, but that does not really make it ok - fixed in the commit below. thanks!